### PR TITLE
added script to change eclipse versions

### PR DESCRIPTION
**Pull request by lhein**
Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [ ] Did you set meaningful commit comments on each commit?
- [ ] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

File: changeEclipse.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (apupier) [inline changeEclipse.sh]: any reasons to let an empty first line?

File: changeEclipse.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# Initial version from lhein, 2018      #
+#########################################
+
+# check parameter count and provide help if count doesn't match
+if [ "$#" -ne 2 ]; then
+    echo "Illegal number of parameters"
+    echo "Please specify the old and the new Eclipse name like this:"
+    echo "      changeVersion.sh <oldEclipseName> <newEclipseName>"
+    echo "Example:"
+    echo "      changeVersion.sh photon photon"
```
Comment (apupier) [inline changeEclipse.sh]: might be best to provide a real exemple changeVersion.sh oxygen photon for instance ;-)

File: changeEclipse.sh (hunk 1 above)
Comment (lhein) [reply changeEclipse.sh]: any reason not to have an empty first line?

File: changeEclipse.sh (hunk 2 above)
Comment (lhein) [reply changeEclipse.sh]: I fear I was victim of my own script testing. Will repair :D

File: changeEclipse.sh (hunk 1 above)
Comment (apupier) [reply changeEclipse.sh]: it's just the first time that I see that so i'm curious if it is a good practice i'm not aware of.
I would think that less code is better.

**Comment by apupier** [comment]
remove "ready for review" as build is in error

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
added PRIO label as I need this script to update all oxygen refs in the codebase to photon

**Pull request merged**